SECURITY.md

```diff
@@ -1,26 +1,30 @@
 # Security Policy
 
-## Supported Versions
-
-To receive fixes for security vulnerabilities it is required to always upgrade to the latest version of Keycloak. 
-See https://www.keycloak.org/downloads for the latest release.
+The Keycloak team takes security very seriously, and aim to resolve issues as quickly as possible. Building secure
+software is a continuous process, and can always be improved. As such we welcome reports on potential security 
+vulnerabilities, as well as suggestions around hardening the software and our process.
 
-Fixes will only be released for previous releases under special circumstances.
+## Reporting a suspected vulnerability
 
-## Reporting a Vulnerability
+It is important that suspected vulnerabilities are disclosed in a responsible way, and are not publicly disclosed until
```

Comment:
```suggestion
We strongly encourage disclosing suspected vulnerabilities responsibly, and only disclose them publicly
```

Reply:
'strongly encourage' is too soft - should be written more as a requirement, rather than a suggestion

Reply:
I don't mind either formulation, though I incline a bit more to the softer variant. Both original formulations here and below look good to me, so feel free to merge it as is.